timesketch/lib/testlib.py

```diff
@@ -82,9 +82,6 @@ def search(self, index, body, size):  # pylint: disable=unused-argument
 class MockDataStore(object):
     """A mock implementation of a Datastore."""
 
-    #List containing event dictionaries
```

Comment:
event_store was moved inside the __init__ method, because otherwise the MockDataStore class definition was changed every time when the event_store was changed.